### PR TITLE
Insert messages from Flask to DB

### PR DESCRIPTION
**Pull request by ChristianPerez34**

### CODE REVIEW

File: resources.py [hunk 1]
```diff
@@ -259,11 +258,15 @@ def get(self, chat_id):
     # @jwt_required
     def post(self, chat_id):
         parser = reqparse.RequestParser()
+        parser.add_argument('uid', help=HELP_TEXT, required=True)
         parser.add_argument('message', help=HELP_TEXT, required=True)
         parser.add_argument('img')
         data = parser.parse_args()
-        message = ChatHandler().insert_chat_message(message='message',
-                                                    img='/images/branding/googlelogo/2x/googlelogo_color_92x30dp.png')
+        if 'img' in data and data['img']:
```
Comment (javierbustillo) [inline resources.py]: img has to be saved as a file in static/img. That is what you save into the DB

**Review by javierbustillo — CHANGES_REQUESTED** [review]

File: resources.py (hunk 1 above)
Comment (ChristianPerez34) [reply resources.py]: Handler should take care of that, however that is only boiler plate code for when the issue concerning messages with images

**Comment by javierbustillo** [comment]
Issue es que si Manuel sube una foto en el happy hour, se rompe

File: resources.py (hunk 1 above)
Comment (ChristianPerez34) [reply resources.py]: What do you mean? The POST request should come with the message and image, afterwards we are to extract the image from the response, store it in the appropriate directory then proceed to store both the message and the image url in the DB

**Comment by javierbustillo** [comment]
El frontend no esta subiendo fotos en el POST

**Review by javierbustillo — APPROVED** [review]

**Pull request merged**